packages/lib/src/types.ts

```diff
@@ -22,14 +24,16 @@ const documentData = z.object({
 })
 export type DocumentData = z.infer<typeof documentData>
 
-const unsignedCreateOpV1 = z.object({
-  type: z.literal('create'),
-  signingKey: z.string(),
-  recoveryKey: z.string(),
-  handle: z.string(),
-  service: z.string(),
-  prev: z.null(),
-})
+const unsignedCreateOpV1 = z
+  .object({
+    type: z.literal('create'),
+    signingKey: z.string(),
+    recoveryKey: z.string(),
+    handle: z.string(),
+    service: z.string(),
+    prev: z.null(),
+  })
+  .strict()
```

Comment:
I don't think we have any invalid legacy create ops in the directory, but I made this strict too for completeness.